.zuul.yaml

```diff
@@ -0,0 +1,8 @@
+
+---
+- project:
+    name: infrawatch/sg-core
+    github-check:
+      jobs:
+        - stf-crc-latest-nightly_bundles
```

Comment:
@elfiesmelfie Does this nightly bundles test do anything involving the change, or is it just a side by-side to help us quickly rule out any non-change-related problems?

Reply:
It is a side-by-side to help us rule out issues. 

Eventually, this will be run periodically, rather than on a change-by-change basis. However, since there's a lot of change for stf-run-ci at the moment, it's not a bad idea to run the job as often as possible to shake out bugs